scalajs/src/main/scala/io/circe/scalajs/package.scala

```diff
@@ -0,0 +1,83 @@
+package io.circe
+
+import cats.data.Xor
+import io.circe.Decoder.Result
+import io.circe.Json._
+import io.circe.parse._
+import scala.scalajs.js
+import scala.scalajs.js.JSConverters.{JSRichGenMap, JSRichGenTraversableOnce}
+import scala.scalajs.js.{SyntaxError, JavaScriptException}
+
+package object scalajs {
+
+  /**
+   * Converts scalajs js.Any to circe Json
+   * @param input
```

Comment:
Do you mind removing the Scaladoc stubs (or filling them out, although in this case I don't think that'd add much value)?